x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.test.ts

```diff
@@ -46,26 +49,78 @@ describe('getTransactionBreakdown', () => {
       }
     });
 
-    expect(response.length).toBe(3);
+    expect(response.kpis.length).toBe(4);
 
-    expect(response.map(breakdown => breakdown.name)).toEqual([
+    expect(response.kpis.map(kpi => kpi.name)).toEqual([
       'app',
-      'mysql',
-      'elasticsearch'
+      'http',
+      'postgresql',
+      'dispatcher-servlet'
     ]);
 
-    expect(response[0]).toEqual({
-      count: 15,
+    expect(response.kpis[0]).toEqual({
       name: 'app',
-      percentage: 2 / 3
+      percentage: 0.5408550899466306
+    });
+
+    expect(response.kpis[2]).toEqual({
+      name: 'postgresql',
+      percentage: 0.047366859295002
     });
+  });
+
+  it('returns a timeseries grouped by type and subtype', async () => {
+    const clientSpy = jest.fn().mockReturnValueOnce(dataResponse);
+
+    const response = await getTransactionBreakdown({
+      serviceName: 'myServiceName',
+      setup: {
+        start: 0,
+        end: 500000,
+        client: { search: clientSpy } as any,
+        config: {
+          get: () => 'myIndex' as any,
+          has: () => true
+        },
+        uiFiltersES: []
+      }
+    });
+
+    const { timeseries_per_subtype: timeseriesPerSubtype } = response;
```

Comment:
Why are we using snake case in the first place? All APM endpoints return camel cased data currently.

Reply:
https://github.com/elastic/kibana/blob/868c7125c64a4b7b25484a307cde4e9aafa7109f/x-pack/legacy/plugins/apm/server/lib/transactions/breakdown/index.ts#L191